### PR TITLE
Html structure

### PR DESCRIPTION
**Pull request by PacoSchwab**
HTML only, please review :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,14 +1,80 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
```
Comment (rabe200) [inline index.html:3]: looks like there are too many heads?

**Review by rabe200 — CHANGES_REQUESTED** [review]
besides of 2 heads alright

File: index.html (hunk 1 above)
Comment (PacoSchwab) [reply index.html:3]: thx for comment. do you mean head or header? I only see one head

Comment (rabe200) [reply index.html:3]: sorry - that was my mistake - it is all okay, I interpreted something the wrong way! all lgtm 👍

**Pull request merged**